upgrade-tests.groovy

```diff
@@ -109,7 +109,7 @@ node('cico-workspace') {
 			timeout(time: 120, unit: 'MINUTES') {
 				upgrade_args = "--test-cephfs=false --test-rbd=true --upgrade-testing=true"
 				if ("${test_type}" == "cephfs"){
-					upgrade_args = "--test-cephfs=true --test-rbd=false --upgrade-testing=true"
+					upgrade_args = "--deploy-timeout=300 --test-cephfs=true --test-rbd=false --upgrade-testing=true"
```

Comment:
this won't be helpful the error we are getting is from the kubernetes server. we may need to handle this error in our e2e code.

Reply:
oh, yes, maybe error handling is not done sufficiently there

Reply:
#1626 could be a better fix